arch/risc-v/include/elf.h

```diff
@@ -103,5 +106,13 @@ struct arch_elfdata_s
 };
 typedef struct arch_elfdata_s arch_elfdata_t;
 
+struct user_pt_regs
+{
+  unsigned long regs[32];
```

Comment:
should we consider this patch: https://github.com/apache/nuttx/pull/12475

Reply:
Yes, I've updated as necessary.